### PR TITLE
Add standalone test

### PR DESCRIPTION
**Pull request by black-adder**
Adding a test for the standalone docker image that creates a trace with via the query service and then retrieves it.

- [x] Rebase with https://github.com/uber/jaeger/pull/124 once it lands

### CODE REVIEW

File: cmd/standalone/main.go [hunk 1]
```diff
@@ -118,6 +119,17 @@ func startCollector(logger *zap.Logger, baseFactory metrics.Factory, memoryStore
 func startQuery(logger *zap.Logger, baseFactory metrics.Factory, memoryStore *memory.Store) {
 	metricsFactory := baseFactory.Namespace("jaeger-query", nil)
 
+	jaegerClientConfig.Configuration{
+		Sampler: &jaegerClientConfig.SamplerConfig{
+			Type:  "const",
+			Param: 1.0,
+		},
+		RPCMetrics: true,
+	}.InitGlobalTracer(
```
Comment (yurishkuro) [inline cmd/standalone/main.go]: what is the reason to init global tracer?

File: cmd/standalone/standalone_test.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+	cli, err := client.NewEnvClient()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	exposedPorts, portBindings, _ := nat.ParsePortSpecs([]string{
+		fmt.Sprintf("%s:%s", queryHostPort, queryPort),
+	})
+
+	resp, err := cli.ContainerCreate(
```
Comment (yurishkuro) [inline cmd/standalone/standalone_test.go]: the amount of dependencies this brings is very unfortunate. We don't use any of that for crossdock tests, why is it necessary here? Can't we run the test via docker-compose?

File: cmd/standalone/main.go (hunk 1 above)
Comment (black-adder) [inline cmd/standalone/main.go]: so the standalone query service is instrumented.

Comment (yurishkuro) [inline cmd/standalone/main.go]: my bad. We should change it to not use global tracer, but to expect tracer as an option.

File: cmd/standalone/standalone_test.go (hunk 2 above)
Comment (black-adder) [inline cmd/standalone/standalone_test.go]: We can... it's just more work... How adamant are you for keeping the number of dependencies low?

Comment (black-adder) [inline cmd/standalone/standalone_test.go]: nvm

**Pull request merged**